### ISSUE TITLE
Mod Incompatibility with Blackgear27's Caves and Cliff Backport

### ISSUE THREAD

**Issue by nerdula** [MaxNeedsSnacks/roadrunner#18]
Strange issue with only Roadrunner and Caves and Cliffs Backport, (Release or built from latest both cause the same issue) on Forge. FML/Minecraft will start to load, and about half way through the progress bar (At registering minecraft:block) the game will freeze for a second, and then crash with no reason and code 0. Even stranger is that both latest.log and debug.log don't show any issues as far as I know from my time playing modded. I don't really know whether this is really a Roadrunner bug or the Caves and Cliffs bug. Sorry if I missed something really obvious.

Versions: Forge 36.2.8, Caves and Cliffs 1.16.5-6.3, Roadrunner 1.16.5-1.3.0 

Latest.log attached below:
[latest.log](https://github.com/MaxNeedsSnacks/roadrunner/files/7357008/latest.log)

**Comment by MaxNeedsSnacks** [comment]
Yeah as you said, this log unfortunately doesn't tell me much of *anything*...

Does the debug.log tell you anything or is there a crash report at least? Otherwise I'll have no idea where to start on a fix for this

**Comment by evanbones** [comment]
I can confirm this at least, but there's no info in the log. I'm like 99% it's a mixin conflict though

**Comment by nerdula** [comment]
Sorry, currently on mobile so I can't send the debug log, but from what I remember, it didn't have any errors either and there wasn't even a crash report, so yeah I will probably agree with evanbones that it's a mixin conflict as mixin seems to be pretty inconsistent with its reports (with my limited knowledge).

**Comment by Reshirou** [comment]
I'm running both and haven't crashed

Beware Caves n Cliffs is almost done with it's total rewrite though, so this won't matter soon anyway unless the rewrite is incompatible

**Comment by nerdula** [comment]
Closing issue found the problem, I've found this isn't a problem with either particular mod, this is a strange issue with the AdoptOpenJDK OpenJ9 VM from https://developer.ibm.com/languages/java/semeru-runtimes/downloads. This issue only occurs when running both mods at the same time. Running either mod individually doesn't cause this issue. Running the normal Minecraft VM does not cause this issue and can be completely resolved via just using the normal VM. My bad for causing any confusion for either MaxNeedsSnacks and BlackGear27.
